projects/freetype2/Dockerfile

```diff
@@ -21,3 +21,10 @@ RUN apt-get install -y make autoconf libtool libarchive-dev
 RUN git clone --depth 1 git://git.sv.nongnu.org/freetype/freetype2.git
 WORKDIR freetype2
 COPY build.sh $SRC/
+
+# Get some files for the seed corpus
+RUN wget https://github.com/adobe-fonts/adobe-variable-font-prototype/releases/download/1.001/AdobeVFPrototype.otf
+
+
+# Put all seed files into $OUT/ftfuzzer_seed_corpus.zip
+RUN zip -j $OUT/ftfuzzer_seed_corpus.zip $(find . -name '*.otf')
```

Comment:
@alex is right: when we run the build in the container we mount build directory to container's '/out'. So whatever you put there disappears after the mount because the build directory doesn't have those files.

I have contemplated to create `$STATICOUT` but I'm afraid that would create more confusion.